src/components/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,105 @@
+import React, { useState, ComponentProps } from 'react';
+import { getCommonProps } from '../../utils';
+import classnames from 'classnames';
+import * as DropdownSelect from '@radix-ui/react-select';
+import ChevronDownIcon from '../../assets/chevronDown.svg?react';
+import ChevronUpIcon from '../../assets/chevronUp.svg?react';
+
+export interface DropdownProps
+  extends Omit<DropdownSelect.SelectProps, 'defaultValue' | 'dir'>,
+    Omit<ComponentProps<'div'>, 'ref'> {
+  /**
+   * All options to be listed in the dropdown
+   */
+  options: {
+    label: string;
+    value: string;
+  }[];
+  /**
+   * Current selected value
+   */
+  value: DropdownSelect.SelectProps['defaultValue'];
+  /**
+   * Pass a call back that triggers once a new value is selected
+   */
+  onValueChange: (value: string) => void;
+  /**
+   * Aria-label for specific dropdown use
+   */
+  label: string;
+}
+
+/**
+ * ## Overview
+ *
+ * Overview of this component
+ *
+ * [Figma Link](https://www.figma.com/design/hMu9IWH5N3KamJy8tLFdyV/EASEL-Compendium%3A-Tokens%2C-Components-%26-Patterns?node-id=10570-6295&t=EnYSS6Hk58EiWaVg-4)
+ *
+ * [Storybook Link](https://phillips-seldon.netlify.app/?path=/docs/components-dropdown--overview)
+ */
+const Dropdown = React.forwardRef<HTMLButtonElement, DropdownProps>(
+  ({ options, value, onValueChange, label, className, id, ...props }, ref) => {
+    const { className: baseClassName, ...commonProps } = getCommonProps({ id }, 'Dropdown');
+    const [isOpen, setIsOpen] = useState(false);
+
+    return (
+      <div className={classnames(baseClassName, className)} id={id} {...commonProps} {...props}>
+        <DropdownSelect.Root
+          defaultValue={value}
+          onValueChange={(value) => {
+            onValueChange(value);
+          }}
+          onOpenChange={setIsOpen}
+        >
+          <DropdownSelect.Trigger className={`${baseClassName}__trigger`} aria-label={label} ref={ref}>
+            <DropdownSelect.Value placeholder={value} />
+            <DropdownSelect.Icon>
+              {<ChevronDownIcon className={isOpen ? `${baseClassName}__trigger-icon-expanded` : ''} />}
```

Comment:
This would be the scenario where we would want to use the classnames packages. 
```suggestion
              {<ChevronDownIcon className={classnames({ [`${baseClassName}__trigger-icon-expanded`] : isOpen })} />}
```